src/pages/iou/request/IOURequestStartPage.tsx

```diff
@@ -108,17 +105,10 @@ function IOURequestStartPage({
     };
 
     const resetIOUTypeIfChanged = useCallback(
-        (newIouType: IOURequestType) => {
-            if (newIouType === previousIOURequestType) {
-                return;
-            }
```

Comment:
`resetIOUTypeIfChanged` is called on first render and this condition should have been kept. If we refresh the page, the iou type won't change thus we should not reinit the money request. (Coming from https://github.com/Expensify/App/issues/40936)